compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -430,7 +431,21 @@ setImports :: [FilePath] -> DynFlags -> DynFlags
 setImports paths dflags = dflags { importPaths = paths }
 
 locateGhcVersionHeader :: IO GhcVersionHeader
-locateGhcVersionHeader = GhcVersionHeader <$> locateRunfiles (mainWorkspace </> "compiler" </> "damlc" </> "ghcversion.h")
+locateGhcVersionHeader = GhcVersionHeader <$> do
+    resourcesDir <- locateRunfiles (mainWorkspace </> "compiler" </> "damlc" </> "ghcversion.h")
+    isDirectory <- doesDirectoryExist resourcesDir
+    let path | isDirectory = resourcesDir </> "ghcversion.h"
```

Comment:
What's the idea behind this flexibility here?

Reply:
It's because `locateRunfiles` sometimes returns the `ghcversion.h` file produced by the bazel rule, or returns the resources directory.

This is super janky and I *think* I can fix it to always return a directory (by asking for "compiler/damlc" instead of the specific file).

I thin we were just using `locatRunfiles` wrongly, here.

Reply:
> somehow necessary because of Windows

has convinced me. 😸